app/view/pages/home/index.js

```diff
@@ -65,14 +69,14 @@ const Home = () => {
               )}>
               <Button
                 mode="outlined"
-                color={'#13a77f'}
+                color="#13a77f"
```

Comment:
Excuse me, but the `#13a77f` color used multiple times. why you didn't put in some config file like `theme.js`?

Reply:
I have to go with a complete solution for theme, it's in my list but at the time I don't want to involve with some more thing 
if it's acceptable for now